Windows tests started to fail

only last week

![image](https://github.com/datalad/datalad-installer/assets/39889/cc338164-c9fc-4341-ba42-54d13d54f2ab)

e.g

```
=========================== short test summary info ===========================
FAILED test/test_install.py::test_install_venv_miniconda_datalad - subprocess.CalledProcessError: Command '['D:\\a\\datalad-installer\\datalad-installer\\.tox\\py\\Scripts\\python.EXE', '-m', 'venv', 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_install_venv_miniconda_da0\\venv']' returned non-zero exit status 1.
FAILED test/test_install.py::test_install_venv_miniconda_conda_env_datalad - subprocess.CalledProcessError: Command '['D:\\a\\datalad-installer\\datalad-installer\\.tox\\py\\Scripts\\python.EXE', '-m', 'venv', 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_install_venv_miniconda_co0\\venv']' returned non-zero exit status 1.
FAILED test/test_install.py::test_install_venv_datalad - subprocess.CalledProcessError: Command '['D:\\a\\datalad-installer\\datalad-installer\\.tox\\py\\Scripts\\python.EXE', '-m', 'venv', 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_install_venv_datalad0\\venv']' returned non-zero exit status 1.
FAILED test/test_install.py::test_install_venv_dev_pip_datalad - subprocess.CalledProcessError: Command '['D:\\a\\datalad-installer\\datalad-installer\\.tox\\py\\Scripts\\python.EXE', '-m', 'venv', 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_install_venv_dev_pip_data0\\venv']' returned non-zero exit status 1.
FAILED test/test_install.py::test_install_miniconda_conda_env_venv_datalad - subprocess.CalledProcessError: Command '['D:\\a\\datalad-installer\\datalad-installer\\.tox\\py\\Scripts\\python.EXE', '-m', 'venv', 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_install_miniconda_conda_e1\\venv']' returned non-zero exit status 1.
======= 5 failed, 95 passed, 10 skipped, 1 xpassed in 608.62s (0:10:08) =======
py: exit 1 (610.86 seconds) D:\a\datalad-installer\datalad-installer> pytest -vv --ci --cov-report=xml test pid=7004
  py: FAIL code 1 (647.67=setup[36.81]+cmd[610.86] seconds)
  evaluation failed :( (651.48 seconds)
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -35,8 +35,11 @@ jobs:
           - '3.11'
           - '3.12'
           - 'pypy-3.8'
-          - 'pypy-3.9'
-          - 'pypy-3.10'
+          # Avoid PyPy 7.3.16, as the tests currently fail on it due to a bug
+          # in it and/or tox: <https://github.com/pypy/pypy/issues/4958>,
+          # <https://github.com/tox-dev/tox/issues/3284>
+          - 'pypy-3.9-v7.3.15'
+          - 'pypy-3.10-v7.3.15'
         toxenv: [py]
         exclude:
           # No older Pythons on arm64 macos-latest

